<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exportui</class>
 <widget class="QDialog" name="Exportui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export Track</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QComboBox" name="exportFromBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="exportFromLabel">
       <property name="text">
        <string>Export From Section:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="exportTypeLabel">
       <property name="text">
        <string>Exporter Type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="exportToBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="exportToLabel">
       <property name="text">
        <string>Export To Section</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="exportTrackLabel">
       <property name="text">
        <string>Export Track</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="exportTrackBox"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="exportTypeBox">
       <item>
        <property name="text">
         <string>NoLimits2 Exporter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tangent Exporter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spline Exporter</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="nl1_frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="nl1_layout">
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="noHeartLineBox">
          <property name="text">
           <string>Export without heartlining the Track</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="exportMPerNodeLabel">
          <property name="text">
           <string>Segment Length (in m)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="segmentLengthBox">
          <property name="minimum">
           <double>0.200000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.200000000000000</double>
          </property>
          <property name="value">
           <double>2.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="relThreshLabel">
          <property name="text">
           <string>Threshold for RelRoll (in Â°)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="relThresBox">
          <property name="minimum">
           <double>75.000000000000000</double>
          </property>
          <property name="maximum">
           <double>90.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
          <property name="value">
           <double>85.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Exportui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Exportui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
